app/electron/main.ts

```diff
@@ -319,6 +344,30 @@ function getDefaultAppMenu(): AppMenu[] {
           role: 'forcereload',
           id: 'original-force-reload',
         },
+        {
+          label: i18n.t('Go To Home'),
+          role: 'homescreen',
+          id: 'original-home-screen',
+          click: () => {
+            mainWindow?.loadURL(startUrl);
+          },
+        },
+        {
+          label: i18n.t('Go Back'),
+          role: 'back',
+          id: 'original-back',
```

Comment:
It would be nice if this and the Go Forward were disabled when there's no way to go back/forward. This has to be done by responding to a webContents event, then fetching the menus by ID and setting them as not enabled.

Reply:
done